engines/bastion_katello/app/assets/javascripts/bastion_katello/organizations/check-current-organization.run.js

```diff
@@ -8,22 +8,10 @@
      *   Checks whether a page requires a current organization to be set and if it does
      *   redirects the user to the Katello 403 page to instruct them to select an organization to proceed.
      */
-    function CheckCurrentOrganization($rootScope, $window, CurrentOrganization) {
-        var fencedPages = [
-            'products',
-            'activation-keys',
-            'environments',
-            'subscriptions',
-            'gpg-keys',
-            'sync-plans',
-            'content-views',
-            'errata',
-            'content-hosts',
-            'host-collections'
-        ];
+    function CheckCurrentOrganization($rootScope, $window, CurrentOrganization, FencedPages) {
 
         $rootScope.$on('$stateChangeStart', function (event, toState) {
-            if (CurrentOrganization === "" && fencedPages.indexOf(toState.name.split('.')[0]) !== -1) {
+            if (CurrentOrganization === "" && FencedPages.list().indexOf(toState.name.split('.')[0]) !== -1) {
```

Comment:
What do you think about adding a method to do this check instead of forcing each use of the code to have to do this? Basically, a function a that can tell you whether or not a page is in the list.